coalib/parsing/DefaultArgParser.py

```diff
@@ -182,18 +182,24 @@ def default_arg_parser(formatter_class=None):
                                    action='store_const',
                                    help=SHOW_BEARS_HELP)
 
-        outputs_group.add_argument('-A',
-                                   '--show-all-bears',
-                                   const=True,
-                                   action='store_const',
-                                   help="Display all bears.")
-
-        SHOW_LANGUAGE_BEARS = ("Display all bears for the given languages.")
         outputs_group.add_argument('-l',
-                                   '--show-language-bears',
+                                   '--filter-language',
                                    nargs='+',
                                    metavar='LANG',
-                                   help=SHOW_LANGUAGE_BEARS)
+                                   help="Filters the output of `--show-bears` "
+                                        "by the given languages.")
+
+        outputs_group.add_argument('-D',
+                                   '--show-description',
+                                   const=True,
+                                   action='store_const',
+                                   help="Shows bear descriptions for the "
+                                        "`--show-bears` output.")
+        outputs_group.add_argument('--show-details',
+                                   const=True,
+                                   action='store_const',
+                                   help='Shows bear details for the '
+                                        '`--show-bears` output.')
```

Comment:
Could you be a little more descriptive here about what details it shows? I couldn't understand right away, plus it's repetitive considering the flag is `--show-details`.